cs464-group-project/src/pages/Players/IndividualPlayer.js

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import axios from 'axios';
+
+export function IndividualPlayer() {
+  const { id } = useParams();
+  const [playerData, setPlayerData] = useState(null);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const response = await axios.get(
+          `https://www.thesportsdb.com/api/v1/json/3/lookupplayer.php?id=${id}`
+        );
+        setPlayerData(response.data.players[0]);
+      } catch (error) {
+        console.error('Error fetching player data:', error);
+      }
+    };
+
+    fetchData();
+  }, [id]);
+
+  return (
+    <div style={{ display: 'flex', margin: '20px' }}>
```

Comment:
Lets try to use bootstrap grids and styling to keep our pages consistent.

Reply:
Changed to bootstrap grids